### PR TITLE
Fixes runed metal and brass sprites

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
fix: Runed metal and brass are no longer invisible.
/:cl:

It seems like someone put these into a new dmi, but for some godforsaken reason they ignored the brass and runed metal objects. Now they actually appear!

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -270,7 +270,7 @@ GLOBAL_LIST_INIT(runed_metal_recipes, list ( \
 	desc = "Sheets of cold metal with shifting inscriptions writ upon them."
 	singular_name = "runed metal sheet"
 	icon_state = "sheet-runed"
-	icon = 'icons/obj/items_and_weapons.dmi'
+	icon = 'icons/obj/stack_objects.dmi'
```
Comment (ChangelingRain) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: this should be inherited, so it should thus be removable

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -322,7 +322,7 @@ GLOBAL_LIST_INIT(brass_recipes, list ( \
 	desc = "Sheets made out of brass."
 	singular_name = "brass sheet"
 	icon_state = "sheet-brass"
-	icon = 'icons/obj/items_and_weapons.dmi'
+	icon = 'icons/obj/stack_objects.dmi'
```
Comment (ChangelingRain) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: this should be inherited, so it should thus be removable

Comment (Xhuis) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: It is not inherited, tiles use `tiles.dmi`

**Comment by AnonymousNow** [comment]
Do these have sprites for stacks with multiple units in them, like other such materials?

**Comment by Xhuis** [comment]
Brass does, but runed metal doesn't.

**Comment by Bawhoppen** [comment]
Sorry I thought I got everything

**Pull request merged**